deep_field_metadetect/gaussmom/gaussmom_core.py

```diff
@@ -0,0 +1,120 @@
+from typing import NamedTuple
+
+import jax
+import jax.numpy as jnp
+import ngmix
+
+
+@jax.tree_util.register_pytree_node_class
+class GaussMomObs(NamedTuple):
```

Comment:
We should use a single observation type for everything. I guess the goal is to move this over to match depp-field mdet later?